File: CursoCsharpUdemy/exer/outros/Exer02.cs
/***********************************
*   2) Faça um programa que:
*   - Leia a cotação do dólar
*   - Leia um valor em dólares
*   - Converta esse valor para Real
*   - Mostre o resultado
************************************/

using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace exer.outros
{
    public class Exer02
    {
        static void Main(string[] args)
        {
            Console.WriteLine("Informe a cotação do dolar atual: ");
            double dollar = double.Parse(Console.ReadLine());

            Console.WriteLine("Informe seu sálario em dolar: ");
            double salaryDollar = double.Parse(Console.ReadLine());

            double salaryReal = salaryDollar * dollar;

            Console.Write("Seu sálario em real será: " + salaryReal.ToString("F2"));
        }
    }
}

File: CursoJava/old/Curso de Java/Exer/src/lista64Exer/Exer02.java
package lista64Exer;
/*************************************************************************
 *  2) Faça um programa que:
 *    - Leia a cotação do dólar
 *    - Leia um valor em dólares
 *    - Converta esse valor para Real
 *    - Mostre o resultado
 *
 *************************************************************************/

import java.util.Locale;
import java.util.Scanner;

public class Exer02 {

    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        System.out.print("Informe a cotação do dolar atualmente: ");
        double quotation = sc.nextDouble();

        System.out.println("Informe seu salário: ");
        double salary = sc.nextDouble();

        double convert = quotation * salary;

        System.out.printf("Seu salário em real, seria R$%.2f%n", convert);

        sc.close();
    }

}
